packages/xstate-inspect/src/serialize.ts

```diff
@@ -20,13 +20,16 @@ export function selectivelyStringify<T extends object>(
   });
 }
 
-export function stringifyState(state: AnyState, replacer?: Replacer): string {
+export function stringifyState(
+  { machine, configuration, ...state }: AnyState,
+  replacer?: Replacer
+): string {
```

Comment:
```suggestion
  stateToStringify: AnyState,
  replacer?: Replacer
): string {
  const { machine: _machine, configuration: _configuration, ...state } = stateToStringify;
```

Alternative approach that doesn't destructure in the args (which might not look good for intellisense)